coalib/bears/Bear.py

```diff
@@ -84,6 +99,13 @@ class Bear(Printer, LogPrinter):
 
     LANGUAGES = ()
     REQUIREMENTS = ()
+    AUTHOR = {}
+    AUTHOR_EMAIL = {}
+    MAINTAINERS = {}
+    MAINTAINERS_EMAILS = {}
+    PLATFORMS = {'any'}
+    LICENSE = ''
```

Comment:
@sils1297 i made all sets besides License. I think that theres no point this should be a set and not a string :)